charts/flagsmith/templates/deployment-api.yaml

```
@@ -82,6 +82,8 @@ spec:
         ports:
         - containerPort: {{ .Values.service.api.port }}
         env: {{ include (print $.Template.BasePath "/_api_environment.yaml") . | nindent 8 }}
+        extraVolumeMounts:
+          {{- toYaml .Values.extraVolumeMounts | default "" | nindent 12 }} 
```

```suggestion
        {{- with .Values.api.extraVolumeMounts }}
        volumeMounts: {{- toYaml . | nindent 8 }}
        {{- end }}
```
I don't think `extraVolumeMounts` is a valid key here in `v1/Pod`.